vaadin-app-layout-flow-parent/vaadin-app-layout-flow-integration-tests/src/main/java/com/vaadin/flow/component/applayout/examples/I18nPage.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2000-2021 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.vaadin.flow.component.applayout.examples;
+
+import com.vaadin.flow.component.applayout.AppLayout;
+import com.vaadin.flow.component.applayout.AppLayout.AppLayoutI18n;
+import com.vaadin.flow.component.html.Div;
+import com.vaadin.flow.component.html.NativeButton;
+import com.vaadin.flow.router.Route;
+
+@Route("vaadin-app-layout/i18n")
+public class I18nPage extends Div {
+    public I18nPage() {
+        AppLayout layout = new AppLayout();
```

Comment:
nit: is this AppLayout left intentionally empty? The page only ever shows the buttons.

<img width="572" alt="Screenshot 2021-12-22 at 11 03 29" src="https://user-images.githubusercontent.com/1222264/147066560-c834aa9f-67bb-44d6-bb1c-2060f4ffd58d.png">

Reply:
Not intentionally, is it critical? I tried to make up a layout, but I don't know where to put i18n control buttons as the layout fills the full width and height of the screen.

Reply:
At the same time, I can't put them inside the layout because the `toggle attached` button removes the layout. 🤔